tests/test_playwright.py

```diff
@@ -60,28 +89,51 @@ def test_breadcrumb_expansion(page: Page, url_base: str) -> None:
 
 
 def test_breadcrumbs_everywhere(
-    sphinx_build_factory, page: Page, url_base: str
+    sphinx_build_factory: Callable, page: Page, url_base: str
 ) -> None:
-    """Test building the base html template and config."""
-    sphinx_build = sphinx_build_factory("breadcrumbs")
+    """Test breadcrumbs truncate properly when placed in various parts of the layout."""
+    site_name = "breadcrumbs"
+    site_path = _build_test_site(site_name, sphinx_build_factory=sphinx_build_factory)
 
-    # Basic build with defaults
-    sphinx_build.build()
-    assert (sphinx_build.outdir / "index.html").exists(), sphinx_build.outdir.glob("*")
-    symlink_path = path_docs_build / "playwright_tests" / "breadcrumbs"
-    symlink_path.parent.mkdir(exist_ok=True)
-    try:
-        symlink_path.symlink_to(sphinx_build.outdir, True)
+    def check_breadcrumb_truncation():
         page.goto(
-            urljoin(url_base, "playwright_tests/breadcrumbs/hansel/gretel/house.html")
+            urljoin(url_base, f"playwright_tests/{site_name}/hansel/gretel/house.html")
         )
         # sidebar should overflow
         text = "In the oven with my sister, so hot right now. Soooo. Hotttt."
         el = page.locator("#main-content").get_by_text(text).last
         assert _is_overflowing(el)
-        # footer containers will never trigger ellipsis overflow because... their min-width is content? TODO
+        # footer containers never trigger ellipsis overflow because min-width is content
         el = page.locator(".footer-items__center > .footer-item")
         assert not _is_overflowing(el)
-    finally:
-        symlink_path.unlink()
-        symlink_path.parent.rmdir()
+
+    _check_test_site(site_name, site_path, check_breadcrumb_truncation)
+
+
+def test_colors(sphinx_build_factory: Callable, page: Page, url_base: str) -> None:
+    """Test that things get colored the way we expect them to.
+
```

Comment:
These seem to be the "light theme" colours; should we be testing for both light and dark?

My gut tells me yes WDYT @drammock @gabalafou ?

Reply:
I think that as long as we know that mode switching works, tests like this one (i.e. "did the CSS selector do what we intended" tests) only need to test one mode.

The exception would be any a11y test that checks for color contrast --- obv. we should run that test in both modes.